packages/ui/src/SearchDialog.tsx

```diff
@@ -39,15 +39,19 @@ export function SearchDialog({ tracks }: { tracks: (Track & { problems: Problem[
   }, [searchTracks, selectedIndex]);
 
   useEffect(() => {
-    const foundTracks = tracks.filter((track) => {
-      return (
-        track.title.toLowerCase().includes(input.toLowerCase()) ||
-        track.description.toLowerCase().includes(input.toLowerCase())
-      );
-    });
-    setSearchTracks(foundTracks);
-    setSelectedIndex(-1);
-  }, [input]);
+    const timeoutId = setTimeout(() => {
```

Comment:
We're not making any network requests here, so debouncing doesn't provide any benefit